introduction.md

```diff
@@ -0,0 +1 @@
+Hey my name is Cody Weaver I am the master dojo so you must respect me!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.